indico/htdocs/js/indico/modules/contributions/common.js

```diff
@@ -193,6 +196,107 @@
         });
     }
 
+    function onSuccessCallback($element, api) {
+        return function (data) {
+
+            function updateContent(data) {
+                if (data) {
+                    api.set('content.text', ajaxifyForm($(data.html)));
+                    if (data.js) {
+                        $('body').append(data.js);
```

Comment:
I meant you could create an `ajaxQBubble`, similar to `ajaxDialog`. That way other code that also uses ajax-based qbubbles wouldn't have to do all the "dirty work".